File: ABC169/c.cpp
#include <iostream>
#include <math.h>

int main() {
    long long a;
    double b;
    std::cin >> a >> b;

    int bx100 = b * 100;
    long double ans = 0;
    modfl(a * bx100 / 100, &ans);

    std::cout << (long long) ans << std::endl;
}

File: ABC169/c.js
function main(input) {
    const [ a, b ] = input.split(' ')
    const bX100 = b.replace('\.', '')
    const answerStrX100 = '' + (a * bX100)
    const ans = answerStrX100.slice(0, -2)
    if (ans == '') {
        console.log(ans)
    } else {
        console.log(ans)
    }
}

// main(require('fs').readFileSync('/dev/stdin', 'utf8'))
main('10 1.11')
